website/docs/how-to/how-to-create-and-manage-user-groups.md

```diff
@@ -0,0 +1,73 @@
+---
+title: How to create and manage user groups
+---
+
+:::info availability User groups are available to Unleash Enterprise users since **Unleash 4.14**. :::
+
+This guide takes you through how use user groups to manage permissions on your projects. User groups allow you to manage large groups of users more easily than assigning roles directly to those users.
+
+## Creating user groups
+
+1. Navigate to groups by using the admin menu (the gear icon) and selecting the groups option.
+
+![A visual representation of the current step: the Unleash Admin UI with the steps highlighted to navigate to groups.](/img/create-ug-step-1.png)
+
+2. Navigate to new group
+
+![A visual representation of the current step: the groups screen with the new group button highlighted.](/img/create-ug-step-2.png).
+
+3. Give the group a name and an optional description and select the users you'd like to be in the group.
+
+![A visual representation of the current step: the new group screen with the users drop down open and highlighted.](/img/create-ug-step-3.png).
+
+4. Review the details of the group. Each group must contain at least one user with the Owner role, the owner(s) will have permissions to change group membership, so remember to include yourself if you want to manage this group later. If you're happy with the group details, click save. The group is now created.
```

Comment:
The details about management and permissions should go in the reference guide, not the how-to guide.

```suggestion
4. Review the details of the group and save them if you're happy.
```

Reply:
Good call!